chatbot.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 /* eslint-disable no-console */
+
+const request = require('request');
+
```

Comment:
Instead of requiring request here and then making an http request for message handling. I would suggest that you create a new method inside api.js (Named something like - parseUserMessage or something) and then simply call it here!